keyboards/7liang/tp32/readme.md

```diff
@@ -0,0 +1,18 @@
+# Tp32
+
+![tp32](https://image.lceda.cn/pullimage/T7Dbrpw8O7JBuVRb7xHR8AVQtfx1hLb8BAVxwFQ2.jpeg)
+
+![tp32](https://image.lceda.cn/pullimage/dqHcyod5ppyrs9JEMZP2Ei6797HqBOQBMNTA6PyP.jpeg)
+
+this is trackpoint 32 keys keyboard
+
+* Keyboard Maintainer: [skyjun](https://github.com/skyjun/tp32)
+* Hardware Supported: pcb, 3d files, thinkpad trackpoint
+* Hardware Availability: [gerber](https://github.com/skyjun/tp32/tree/main/gerber), [stl](https://github.com/skyjun/tp32/tree/main/stl), [at OSHWHUB](https://oshwhub.com/pkerr/keyborard-juk-32-rev-a)
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```